src/helix-ui/styles/components/radios.less

```diff
@@ -11,6 +11,10 @@ hx-radio {
   width: 1rem;
 }
 
+hx-radio-set {
+  display: block;
+}
```

Comment:
Can we migrate this change to #491? Let's keep this PR about the `docs/` directory.